bin/webpack.js

```diff
@@ -482,6 +484,19 @@ For more information, see https://webpack.js.org/api/cli/.`);
 					);
 				} else if (stats.hash !== lastHash) {
 					lastHash = stats.hash;
+					if (stats.compilation && stats.compilation.errors.length !== 0) {
```

Comment:
`&& outputOptions.entry === undefined`

Reply:
@ev1stensberg I would like to get some context on this comment ```Left a comment. Also, @sokra do you handle default setting in webpack internals? I think the CLI should do that. Sean did a PR but can't find the changes did there.```. What I can make of it is this ```if no option is provided then webpack-cli should throw an error and webpack should not be called```. Is my understanding correct ?